gpt_playground/settings.py

```diff
@@ -472,7 +473,7 @@
 # text-neutral text-error (pipeline run table styles here for tailwind build)
 DJANGO_TABLES2_ROW_ATTRS = {
     "class": "border-b border-gray-200 hover:bg-gray-100",
-    "id": lambda record: f"record-{record.id}",
+    "id": lambda record: f"record-{record.id}" if hasattr(record, "id") else "",
```

Comment:
when does this fail?

Reply:
It fails because the records I was passing in to the table were dicts, not objects. I've updated this too.